src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -54,6 +55,12 @@ internal abstract partial class DefinitionItem
         /// </summary>
         public ImmutableDictionary<string, string> Properties { get; }
 
+        /// <summary>
+        /// Additional diplayable properties that can be attached to the definition for clients that want to
+        /// display additional data.
+        /// </summary>
+        public ImmutableArray<FindUsageProperty> DisplayableProperties { get; }
```

Comment:
consider rename to use `Properties`.

Reply:
Yeah, naming is tricky here...

DefinitionItem already has a `Properties` property that contains information that is not to be displayed (`RQName1` and `RQName2`).  

The name `DisplayableProperties` was used to differentiate the custom columns that can be displayed (`Containing Type` and `Containing Member`).